### PR TITLE
Request to close issue 86

### PR DESCRIPTION
**Pull request by vinhn23**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [ ] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: data-raw/rs_supprs_onco_imwg.R [hunk 1]
```diff
@@ -0,0 +1,123 @@
+# Packages: ---------------------------------------------------------------
```
Comment (manciniedoardo) [inline data-raw/rs_supprs_onco_imwg.R:1]: Please can you use the same header style as for other programs in `data_raw`? Below I include the one for `ae` as an example:
```
# Datasets: ae, suppae
# Description: Standard AE, SUPPAE datasets from CDISC pilot study

# Load libraries -----
library(dplyr)
library(metatools)
library(haven)
library(admiral)

# Create ae, suppae ----
...
```

File: data-raw/rs_supprs_onco_imwg.R [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+library(dplyr)
+
+# Tribble data: -----------------------------------------------------------
```
Comment (manciniedoardo) [inline data-raw/rs_supprs_onco_imwg.R:4]: Please could you shorten section labels to four dashes (package convention)

File: data-raw/rs_supprs_onco_imwg.R [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    STUDYID = "CDISCPILOT01",
+    RDOMAIN = "RS",
+    IDVAR = "RSSEQ",
+    QORIG = "CRF",
+    IDVARVAL=as.character(IDVARVAL)
+  ) %>%
+  relocate(
+    STUDYID, RDOMAIN, USUBJID, IDVAR, IDVARVAL, QNAM, QLABEL, QVAL, QORIG
+  )
+
```
Comment (manciniedoardo) [inline data-raw/rs_supprs_onco_imwg.R:122]: Please could you label the dataset?  See [here](https://github.com/pharmaverse/pharmaversesdtm/blob/cff736346da2b636ccd76814441d7c977aa2ba97/data-raw/dm.R#L15) as an example

File: data-raw/rs_supprs_onco_imwg.R [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+usethis::use_data(supprs_onco_imwg, overwrite = TRUE)
```
Comment (manciniedoardo) [inline data-raw/rs_supprs_onco_imwg.R:123]: Please could you label this section?

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
Thanks for this! Along with my above review please could you:
- Add an entries to `R/data.R` for `rs_onco_imwg` and `supprs_onco_imwg` (this will fix some of the failing checks)
- Address failing `lintr` and `styler` checks
- Ensure you add a changelog entry in `NEWS.md` (for pharmaversesdtm 1.0)

File: data-raw/rs_supprs_onco_imwg.R (hunk 1 above)
Comment (bundfussr) [reply data-raw/rs_supprs_onco_imwg.R]: Updated

File: data-raw/rs_supprs_onco_imwg.R (hunk 2 above)
Comment (bundfussr) [reply data-raw/rs_supprs_onco_imwg.R]: Updated

File: data-raw/rs_supprs_onco_imwg.R (hunk 4 above)
Comment (bundfussr) [reply data-raw/rs_supprs_onco_imwg.R:128]: Done

**Comment by bundfussr** [comment]
> Thanks for this! Along with my above review please could you:
> 
> * Add an entries to `R/data.R` for `rs_onco_imwg` and `supprs_onco_imwg` (this will fix some of the failing checks)
> * Address failing `lintr` and `styler` checks
> * Ensure you add a changelog entry in `NEWS.md` (for pharmaversesdtm 1.0)

@manciniedoardo , thanks for the review. I have done the updates because @vinhn23 is out of office.

**Review by manciniedoardo — APPROVED** [review]

**Comment by manciniedoardo** [comment]
> > Thanks for this! Along with my above review please could you:
> > 
> > * Add an entries to `R/data.R` for `rs_onco_imwg` and `supprs_onco_imwg` (this will fix some of the failing checks)
> > * Address failing `lintr` and `styler` checks
> > * Ensure you add a changelog entry in `NEWS.md` (for pharmaversesdtm 1.0)
> 
> @manciniedoardo , thanks for the review. I have done the updates because @vinhn23 is out of office.

cheers @bundfussr all good now.

**Review by kaz462 — APPROVED** [review]
